src/tests/operate.test.js

```diff
@@ -0,0 +1,11 @@
+import operate from '../logic/operate';
+
+describe('mathematical operations', () => {
+  test('Add two numbers', () => {
+    expect(operate(5, 10, '+')).toBe('15');
+  });
+
+  test('Subtract two numbers', () => {
+    expect(operate(20, 10, '-')).toBe('10');
+  });
```

Comment:
- [ ] As with the calculate function, please cover all the functionalities and scenarios of the `operate` function. Or as much as you can think of. (Every operation, forbidden operations as division or modulus by 0, what happens if strings are sent? etc.